### PR TITLE
XW-3202 | Restyling Forum activity module and Related threads module

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-3202
https://wikia-inc.atlassian.net/browse/XW-3216

@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/Forum/templates/Forum_forumActivityModule.php [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+		<li class="activity-item">
+			<?php $username = User::isIP( $value['display_username'] ) ? wfMessage( 'oasis-anon-user' )->escaped() : htmlspecialchars( $value['display_username'] ); ?>
+
+			<a class="activity-avatar" href="<?= $value['user']->getUserPage()->getFullUrl() ?>">
+				<img class="wds-avatar" src="<?= AvatarService::getAvatarUrl( $value['user']->getName(), 30 ) ?>" data-tracking="activity-avatar" title="<?= $username ?>" />
+			</a>
+
+			<div class="activity-info">
+				<div class="page-title"><!--
+					--><a href="<?= $value['wall_message']->getMessagePageUrl( true ); ?>" title="<?= htmlspecialchars( $value['metatitle'] ); ?>" data-tracking="activity-title"><?= htmlspecialchars( $value['metatitle'] ); ?></a><!--
```
Comment (kvas-damian) [inline extensions/wikia/Forum/templates/Forum_forumActivityModule.php]: For HTML attibutes we have `Sanitizer::encodeAttribute()` we could us it here.

File: extensions/wikia/Forum/css/ForumTag.scss [hunk 2]
```diff
@@ -1,67 +1,86 @@
...
+		height: 30px;
+		width: 30px;
 	}
-	em {
-		overflow: hidden;
-		text-overflow: ellipsis;
-		a {
+
+	.activity-item {
+		@include flexbox();
```
Comment (kvas-damian) [inline extensions/wikia/Forum/css/ForumTag.scss]: Do we need flexbox here?

File: extensions/wikia/Forum/css/ForumTag.scss [hunk 3]
```diff
@@ -1,67 +1,90 @@
...
+		margin: 0 0 8px;
 	}
-	em {
-		overflow: hidden;
-		text-overflow: ellipsis;
-		a {
+
+	.edit-info {
+		color: rgba($color-text, .75);
+		font-size: 12px;
```
Comment (kvas-damian) [inline extensions/wikia/Forum/css/ForumTag.scss]: Please use DS variables for font-sizes and line-heights.

File: extensions/wikia/Forum/templates/Forum_forumActivityModule.php [hunk 4]
```diff
@@ -1,16 +1,24 @@
...
-				<?= wfMessage( $value['is_reply'] ? 'forum-activity-module-posted' : 'forum-activity-module-started' )->rawParams( $user, $time )->escaped(); ?>
+		<li class="activity-item">
+			<?php $username = User::isIP( $value['display_username'] ) ? wfMessage( 'oasis-anon-user' )->escaped() : htmlspecialchars( $value['display_username'] ); ?>
+
+			<a class="activity-avatar" href="<?= $value['user']->getUserPage()->getFullUrl() ?>">
+				<img class="wds-avatar" src="<?= AvatarService::getAvatarUrl( $value['user']->getName(), 30 ) ?>" data-tracking="activity-avatar" title="<?= $username ?>" />
+			</a>
+
+			<div class="activity-info">
+				<div class="page-title"><!--
```
Comment (latata) [inline extensions/wikia/Forum/templates/Forum_forumActivityModule.php]: I'd add a comment why we add an empty comment here it.

**Review by latata — APPROVED** [review]

**Review by weronika — APPROVED** [review]

**Pull request merged**